tests/components/media_player/test_samsungtv.py

```diff
@@ -291,6 +292,32 @@ def test_media_previous_track(self):
         self.device.media_previous_track()
         self.device.send_key.assert_called_once_with("KEY_REWIND")
 
+    async def test_play_media(self):
```

Comment:
Class based unittest tests don't support coroutines as test methods. You need to convert them to standalone pytest test functions. Look at other async tests for examples.

Reply:
👍 thanks. I could tell there was something different about them, but couldn't figure out where the disconnect was.